From fde6b526a9ee0a20d70e992e67ef79f2dfe42362 Mon Sep 17 00:00:00 2001
From: Guru Das Srinagesh <linux@gurudas.dev>
Date: Tue, 21 Jul 2026 00:55:30 -0700
Subject: [PATCH 06840/15333] samples: rust_dma: use vertical import style

Convert `use` imports to vertical layout for better readability and
maintainability.

Signed-off-by: Guru Das Srinagesh <linux@gurudas.dev>
Link: https://patch.msgid.link/20260721-rfl-vert-imp-v2-1-71ce0df519f2@gurudas.dev
Signed-off-by: Danilo Krummrich <dakr@kernel.org>
---
 samples/rust/rust_dma.rs | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/samples/rust/rust_dma.rs b/samples/rust/rust_dma.rs
index b629acc6d915..3b640b2d15cd 100644
--- a/samples/rust/rust_dma.rs
+++ b/samples/rust/rust_dma.rs
@@ -17,10 +17,14 @@
         io_read,
         Io, //
     },
-    page, pci,
+    page,
+    pci,
     prelude::*,
-    scatterlist::{Owned, SGTable},
-    sync::aref::ARef,
+    scatterlist::{
+        Owned,
+        SGTable, //
+    },
+    sync::aref::ARef, //
 };
 
 #[pin_data(PinnedDrop)]
-- 
2.55.0

